R/readme.R

```diff
@@ -33,7 +33,8 @@ use_readme_rmd <- function(base_path = ".") {
   )
   use_build_ignore("^README-.*\\.png$", escape = FALSE, base_path = base_path)
 
-  if (uses_git(base_path) && !file.exists(base_path, ".git", "hooks", "pre-commit")) {
+  if (uses_git(base_path) &&
+      !file.exists(file.path(base_path, ".git", "hooks", "pre-commit"))) {
```

Comment:
Maybe it would be better to create a function for constructing this url? I presume `use_git_hook` also has to do the same computation.  Then you can test that function